src/portal/azext_portal/tests/latest/test_portal_scenario.py

```diff
@@ -0,0 +1,62 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure_devtools.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import JMESPathCheck
+from azure.cli.testsdk import ScenarioTest
+from azure.cli.testsdk import ResourceGroupPreparer
+
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class PortalScenarioTest(ScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='cli_test_portal_testRG'[:9], key='rg')
+    def test_portal(self, resource_group):
+
+        self.kwargs.update({
+            'testDashboard': self.create_random_name(prefix='cli_test_dashboards'[:9], length=24)
+        })
+
+        self.cmd('az portal dashboard create '
+                 '--location "eastus" '
+                 '--input- "src/portal/azext_portal/tests/latest/properties.json" '
+                 '--tags aKey=aValue anotherKey=anotherValue '
+                 '--name "{testDashboard}" '
+                 '--resource-group "{rg}"',
+                 checks=[JMESPathCheck('name', self.kwargs.get('testDashboard', ''))])
```

Comment:
I am not familiar with how these checks work, are these checking that the name of the returned value matches 'testDashboard', or is this a check on the request?

Reply:
If this checks the output, it would be nice to have a check on each of the properties, rather than just one - for create, for update, and for show.  This essentially verifies that the api is modeled correctly

Reply:
> I am not familiar with how these checks work, are these checking that the name of the returned value matches 'testDashboard', or is this a check on the request?

Checks the name in returned value

Reply:
> If this checks the output, it would be nice to have a check on each of the properties, rather than just one - for create, for update, and for show. This essentially verifies that the api is modeled correctly

Good point. I will change to check every property in dashboard